CHANGELOG.md

```diff
@@ -55,6 +55,7 @@ and includes an additional section for migration notes.
     - If preserving a database from a previous version of Orca, set to disaster_recovery.
   - rds_security_group_id (Requires a value. Set in terraform.tfvars to the Security Group ID of your RDS Database's Security Group. Output from Cumulus' RDS module as `security_group_id`)
   - vpc_endpoint_id
+- Adjust workflows/step functions for copy_to_glacier and request_files. `multipart_chunksize_mb` argument in `task_config` is now the Cumulus standard of `s3MultipartChunksizeMb`.
```

Comment:
For migration I don't believe we need to mention request_files. This is something we maintain internally and unlike `copy_to_glacier` users typically do not deploy this Lambda as part of their unique workflow.

Reply:
Removed